src/avram/save_operation.cr

```diff
@@ -33,7 +35,8 @@ abstract class Avram::SaveOperation(T)
 
   @record : T?
   @params : Avram::Paramable
-  getter :record, :params
+  property :record
```

Comment:
hmmm.  I see why you did this but should the be a part of the public api?

Reply:
@wontruefree you mean maybe it should be like `public getter` and `private setter`? I could see something like that

Reply:
Good point. _probably_ not, or at the very least should not be documented